openapi-annotation-processor/src/main/kotlin/io/javalin/openapi/processor/utils/TypesUtils.kt

```diff
@@ -89,6 +89,7 @@ internal object TypesUtils {
         val model = typeMirror.toModel() ?: return ""
 
         return when {
+            model.type == ARRAY && model.simpleName == "Byte" -> "application/octet-stream"
             model.type == ARRAY -> "application/json"
             model.simpleName == "String" -> "text/plain"
             model.simpleName == "ByteArray" || model.simpleName == "[B" -> "application/octet-stream"
```

Comment:
I think we can just replace "ByteArray" condition with the new one and move it above